source/derived.tex

```diff
@@ -800,7 +800,7 @@
 \pnum
 \indextext{definition!virtual~function}%
 A virtual function declared in a class shall be defined, or declared
-pure~(\ref{class.abstract}) in that class, or both; but no diagnostic is
+pure~(\ref{class.abstract}) in that class, or both; no diagnostic is
```

Comment:
The "pure" thing is explained a bit more in [class.abstract]. Nothing in the normative text suggests a function can be abstract.